build/Linux.dockerfile

```diff
@@ -1,42 +1,60 @@
 FROM ua-build-base
 
-RUN wget -O /builds/worker/fetches/binutils.tar.xz https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.cache.level-3.toolchains.v3.linux64-binutils.latest/artifacts/public/build/binutils.tar.xz && \
+ARG IPFS_GATEWAY=https://cloudflare-ipfs.com
+
+ENV MOZ_AUTOMATION_PACKAGE_TESTS=1
```

Comment:
why this is needed? Shouldn't it be put in mozconfig or branding configure.sh?

Reply:
This comes from the taskcluster definition. We import the ENV variables from there.

Reply:
I don't think it is necessary in our case and can potentially increase build times.